3ds2-example/frontend/vite.config.ts

```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from "vite";
+import preact from "@preact/preset-vite";
+
+/**
+ * Proxy API calls from client to backend
+ */
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [preact()],
+  server: {
+    host: true,
+    port: 3020,
```

Comment:
Change to 8080 as a default port (unless this is vite-specific and users are used to 3020

We'll have to update the README to tell folks to whitelist port:8080 in their Customer Area for the drop-in to load

Reply:
✅